packages/x-components/tests/e2e/common/common-steps.spec.ts

```diff
@@ -49,6 +49,11 @@ When('close modal button is clicked', () => {
   cy.getByDataTest('close-main-modal').click();
 });
 
+// Brand Recommendation
+Given('brand recommendations are visible', () => {
+  cy.getByDataTest('query-preview-item').should('be.visible');
```

Comment:
Considering that the `query-preview-item` belongs to the `QueryPreviewList` component, which can be used in several places across the web, and to render other things apart from the brand recommendations, should we add an specific `brand-recommendation` data test id?